Javascript/homework/responses/2.1-css-linking.md

```diff
@@ -4,7 +4,15 @@
 
 This week, we're going to take the HTML and JavaScript code we wrote last week and style our page using CSS to make it more aesthetically pleasing.  
 
-- Copy your index.html file into a new html file in your week 2 homework folder.
+- First you're going to create a new folder called week 2 within your orginal directory. You should now have two folders called "Week 1" and "Week 2"
```

Comment:
I'd recommend numbering these steps and using imperative words ("first, create" instead of "first you're going to") since they're easier to follow (recipes and instructions in general usually do this).